testdata/golang/light/equilibriumIndices.go

```diff
@@ -0,0 +1,16 @@
+package light
+
+func equilibriumIndices(s []int32) (eq []int) {
+	var r, l int64
+	for _, el := range s {
+		r += int64(el)
+	}
+	for i, el := range s {
+		r -= int64(el)
+		if l == r {
+			eq = append(eq, i)
+		}
+		l += int64(el)
+	}
+	return
```

Comment:
Please just use `int` instead of `int32` and `int64`.

Reply:
Please make the variable names a bit more understandable. You can check the Java examples for that.